flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -486,6 +488,47 @@ public void testReadBufferResponseAfterRemovingChannel() throws Exception {
 		testReadBufferResponseWithReleasingOrRemovingChannel(true, false);
 	}
 
+	@Test
+	public void testRemoteInputChannelOnBufferException() throws Exception {
```

Comment:
nit rename to:
`testDoNotFailHandlerOnSingleChannelFailure()`
or
`testDoNotFailHandlerOnSingleChannelOnBufferFailure()`